scripts/tests/test_copydocs.py

```diff
@@ -39,7 +39,7 @@ def get(self, key, revision=None):
         else:
             return revisions.get(revision, None)
 
-    def get_many(self, keys):
+    def get_many(self, keys, max_length = 100):
```

Comment:
```suggestion
    def get_many(self, keys, max_length=500):
```
This `get_many` signature should match that of openlibrary/api.py's `get_many`.